packages/appstore/src/components/elements/text/balance-text.tsx

```diff
@@ -16,7 +16,7 @@ type BalanceTextProps = {
 };
 
 const BalanceText = ({ balance, currency, size = 'm', underline_style = 'none' }: BalanceTextProps) => {
```

Comment:
```suggestion
const BalanceText = observer(({ balance, currency, size = 'm', underline_style = 'none' }: BalanceTextProps) => {
```

We need to add observer here.

Reply:
implemented `export default observer(BalanceText);`

Reply:
It should be on the assignment part not here.
Please add this as suggested in the code change.
According to the documentation, [https://mobx.js.org/react-integration.html#always-read-observables-inside-observer-components](http://url/). Its to reduce re-renders and optimising the performance
We will lose the component type, That's why we need to wrap the component definition itself with observer instead of the reference to the component definition 🙇🏻